packages/styles/table.css

```diff
@@ -2,7 +2,6 @@
   --table-header-text-color: var(--gray-90);
   --table-header-background-color: var(--gray-20);
   --table-header-sorting-background-color: #caddec;
-  --table-header-sorting-text-color: var(--gray-90);
```

Comment:
This is the same color as `--table-header-text-color`. The sorting color does not change.